app/src/main/java/ru/coderedwolf/wordlearn/di/DatabaseModule.kt

```diff
@@ -0,0 +1,68 @@
+package ru.coderedwolf.wordlearn.di
+
+import android.content.Context
+import androidx.room.Room
+import dagger.Module
+import dagger.Provides
+import ru.coderedwolf.wordlearn.database.DataBase
+import ru.coderedwolf.wordlearn.database.mapper.CategoryMapper
+import ru.coderedwolf.wordlearn.database.mapper.PhraseMapper
+import ru.coderedwolf.wordlearn.database.mapper.PhraseTopicMapper
+import ru.coderedwolf.wordlearn.database.mapper.WordMapper
+import javax.inject.Singleton
+
+@Module
+object DatabaseModule {
```

Comment:
Are you sure, that we'll have a profit if we separete DataBase on many Dao dependencies?

Reply:
Yes, I am, cause we will provide only needed daos